frontend/src/routes/index.ts

```diff
@@ -43,6 +44,11 @@ const routes: Routes = {
     path: '/maps',
     title: 'Map Overview',
   },
+  [Pages.Login]: {
+    component: Login,
+    path: '/login',
```

Comment:
Why make it configurable if it is hard-coded after all. VITE_REDIRECT_URI can probably be removed, too. @Bushuo already worked on an issue where such a useless config was removed.

Reply:
Yes, correct.
In fact the whole route, component and env variable is rendered useless with the recent changes.
I will remove it.